.circleci/config.yml

```diff
@@ -1,14 +1,25 @@
-version: 2
+version: 2.1
 jobs:
   lint:
     docker:
       - image: golangci/golangci-lint:v1.23.8-alpine
     steps:
       - checkout
       - run: golangci-lint run
-  test:
+  verify-build:
+    parameters:
+      os:
+        type: string
+      arch:
+        type: string
     docker:
       - image: circleci/golang:1.13
+    steps:
+      - checkout
+      - run: GOOS=<< parameters.os >> GOARCH=<< parameters.arch >> go build ./cmd/secrethub
+  test:
+    docker:
+      - image: circleci/golang:1.14
```

Comment:
There's now multiple Go versions, let's pick one.

Reply:
That was a result of the merge. Well spotted.